workspaces/ui-v2/src/contexts/analytics/implementations/localCliAnalytics.ts

```diff
@@ -60,6 +60,17 @@ export const initialize: AnalyticsStoreProps['initialize'] = async (
     });
     Sentry.setUser({ id: metadata.clientAgent });
   }
+
+  // @ts-ignore
+  if (window.Intercom && appConfig.analytics.intercomAppId) {
+    // @ts-ignore
```

Comment:
Throw this in the top of this file - this should fix the ts ignore things
```
declare global {
  interface Window {
    Intercom?: () => any;
  }
}
```

Reply:
On it